.pre-commit-config.yaml

```diff
@@ -7,6 +7,7 @@ repos:
     rev: 4.3.21
     hooks:
       - id: isort
+        args: ['--virtual-env=.venv']
```

Comment:
ah!  there is a way to set arguments!

Reply:
Yeah! It took me a while to figure out the right one.